opsdroid/message.py

```diff
@@ -21,11 +22,30 @@ def __init__(self, text, user, room, connector):
         self.regex = None
         self.responded_to = False
 
+    async def thinking_delay(self, seconds=0):
+        """Makes opsdroid wait x-seconds before responding."""
+        opsdroid = get_opsdroid()
+
+        if 'thinking-delay' in opsdroid.config:
+            seconds = opsdroid.config.get('thinking-delay')
+
+        if isinstance(seconds, list):
+            seconds = randrange(seconds[0], seconds[1])
+        sleep(seconds)
```

Comment:
This should be `asyncio.sleep()` not `time.sleep()` or it will block the thread.